tests/aws/templates/cfn_lambda_logging_config.yaml

```diff
@@ -0,0 +1,53 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  FunctionName:
+    Type: String
+
+Resources:
+  MyFnServiceRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+        Version: "2012-10-17"
+      ManagedPolicyArns:
+        - Fn::Join:
+            - ""
+            - - "arn:"
+              - Ref: AWS::Partition
+              - :iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+  LambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Ref FunctionName
+      Code:
+        ZipFile: |
+          exports.handler = function(event) {
+            return {
+              statusCode: 200,
+              body: "Hello, World!" 
+            };
+          };
+      Role:
+        Fn::GetAtt:
+          - MyFnServiceRole
+          - Arn
+      Handler: index.handler
+      Runtime: nodejs18.x
```

Comment:
nit: unifying the runtimes (e.g., Python 3.12) could potentially speed up test execution if we don't need to pull many different runtimes. I wouldn't recommend `nodejs18.x`, which will get deprecated later this year (~sept).

Reply:
Good catch, updated that 💯